proto/composerapis/astria/auctioneer/v1alpha1/bundle.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package astria.auctioneer.v1alpha1;
+
+import "astria/execution/v1alpha2/execution.proto";
+import "astria/sequencerblock/v1alpha1/optimistic_block.proto";
+import "google/protobuf/empty.proto";
+
+// Information for the bundle submitter to know how to submit the bundle.
+// The fee and base_sequencer_block_hash are not necessarily strictly necessary
+// it allows for the case where the server doesn't always send the highest fee
+// bundles after the previous but could just stream any confirmed bundles.
+message Bundle {
+  // The fee that can be expected to be received for submitting this bundle.
+  // This allows the bundle producer to stream any confirmed bundles they would be ok
+  // with submitting. Used to avoid race conditions in received bundle packets. Could
+  // also be used by a bundle submitter to allow multiple entities to submit bundles.
+  uint64 fee = 1;
+  // The byte list of transactions to be included.
+  repeated bytes transactions = 2;
+  // The base_sequencer_block_hash is the hash from the base block this bundle
+  // is based on. This is used to verify that the bundle is based on the correct
+  // Sequencer block.
+  bytes base_sequencer_block_hash = 3;
+  // The hash of previous rollup block, on top of which the bundle will be executed as ToB.
+  bytes prev_rollup_block_hash = 4;
+}
+
+message StreamExecuteOptimisticBlockRequest {
+  astria.sequencerblock.v1alpha1.BaseBlock base_block = 1;
+}
+
+message StreamExecuteOptimisticBlockResponse {
+  astria.execution.v1alpha2.Block block = 1;
+}
+
+message StreamBundlesRequest {
+  google.protobuf.Empty empty = 1;
```

Comment:
I don't think with the message you are required to have empty take up a message field can just use an empty message